fecgov/openFEC
# Hotfix/disable newrelic

## Summary (required)

Remove the NEW RELIC python agent from openFEC repo. 

- Resolves https://github.com/fecgov/fec-accounts/issues/201

_Include a summary of proposed changes._

1. Remove all new relic references from :
    - dev/stage/prod manifest files
    - requirements.txt
    - rest.py
2. Rename the NEW_RELIC_APP_NAME environment variable to APP-NAME
    - webservices/tasks/utils.py

## How to test the changes locally
- Deploy this hotfix branch to stage
- Monitor the api, celery-worker and kabana logs
- Test downloads/export functinoality 
- Test legal search, candidate or committee search, should get results back.

## Impacted areas of the application
List general components of the application that this PR will affect:
-  www.fec.gov
